examples/plot_spm_auditory.py

```diff
@@ -1,37 +1,116 @@
+each single-voxel.
+
+The dataset comes from experiment conducted at the FIL by Geriant Rees
+under the direction of Karl Friston. It is provided by FIL methods
+group which develops the SPM software.
+
+According to SPM documentation, 96 acquisitions were made (RT=7s), in
+blocks of 6, giving 16 42s blocks. The condition for successive blocks
+alternated between rest and auditory stimulation, starting with rest.
+Auditory stimulation was bi-syllabic words presented binaurally at a
+rate of 60 per minute. The functional data starts at acquisiton 4,
+image fM00223_004.
+
+The whole brain BOLD/EPI images were acquired on a modified 2T Siemens
+MAGNETOM Vision system. Each acquisition consisted of 64 contiguous
+slices (64x64x64 3mm x 3mm x 3mm voxels). Acquisition took 6.05s, with
+the scan to scan repeat time (RT) set arbitrarily to 7s.
+
+
+This analyse described here is performed in the native space, on the
+original EPI scans without any spatial or temporal preprocessing.
+(More sensitive results would likely be obtained on the corrected,
+spatially normalized and smoothed images).
+
+
+
+
+To run this example, you must launch IPython via ``ipython
+--matplotlib`` in a terminal, or use the Jupyter notebook.
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
 
-The script defines a General Linear Model and fits it to fMRI data.
 
-Author: Bertrand Thirion, dohmatob elvis dopgima, 2015
 """
 
-import os
-import numpy as np
-import pandas as pd
 
-import nibabel as nib
-from nibabel import Nifti1Image
-from nilearn.plotting import plot_stat_map, show
-from nilearn.image import mean_img
 
-from nistats.first_level_model import FirstLevelModel
-from nistats.datasets import fetch_spm_auditory
 
-#########################################################################
-# Prepare spm auditory data
-# ----------------------------------
-# Fetch the dataset
+################################################################################
+# Retrieving the data
+# -------------------
+# 
+# .. note:: In this tutorial, we load the data using a data downloading function.
+#           To input your own data, you will need to pass a list of paths to your own files.
+ 
+
+from nistats.datasets import fetch_spm_auditory
 subject_data = fetch_spm_auditory()
-fmri_img = nib.concat_images(subject_data.func)
-fmri_img = Nifti1Image(fmri_img.get_data(), fmri_img.affine)
-#########################################################################
-# compute background image unto which activation will be projected
+
+
+################################################################################
+# We can list the filenames of the images and display them:
+
+subject_data.func
+from nilearn.plotting import plot_stat_map, plot_anat, plot_img, show
+
+plot_img(subject_data.func[0])
+plot_anat(subject_data.anat)
+show()
+
+################################################################################
+# .. figure:: ../_images/sphx_glr_plot_spm_auditory_001.png
```

Comment:
figure code is created by show, so this does not display correctly, please eliminate all .. figure commands